### PR TITLE
Added support for volumes

### PR DESCRIPTION
**Pull request by chrisdebruin**

### CODE REVIEW

File: lib/mrsk/configuration.rb [hunk 1]
```diff
@@ -61,7 +60,6 @@ def traefik_hosts
     roles.select(&:running_traefik?).flat_map(&:hosts)
   end
 
-
```
Comment (dhh) [inline lib/mrsk/configuration.rb:64]: Extra spaces are intentional to separate sections of related methods.

File: lib/mrsk/configuration.rb [hunk 2]
```diff
@@ -87,6 +84,12 @@ def env_args
     end
   end
 
+  def volumes
+    return unless config.volumes.present?
```
Comment (dhh) [inline lib/mrsk/configuration.rb:88]: Turn into full conditional, like the other usage, so `if config.volumes.present?`.

File: lib/mrsk/configuration.rb (hunk 1 above)
Comment (chrisdebruin) [reply lib/mrsk/configuration.rb]: alright

File: lib/mrsk/configuration.rb (hunk 2 above)
Comment (chrisdebruin) [reply lib/mrsk/configuration.rb]: done

File: lib/mrsk/configuration.rb [hunk 3]
```diff
@@ -87,6 +87,14 @@ def env_args
     end
   end
 
+  def volumes
```
Comment (dhh) [inline lib/mrsk/configuration.rb:90]: This actually needs to be `volume_args` to match the others.

**Pull request merged**